site/dat/docs/JMeter.md

```diff
@@ -165,7 +165,11 @@ scenarios:
     - path: path/to/another.csv  # this is full form, path option is required
       delimiter: ';'  # CSV delimiter, auto-detected by default
       quoted: false  # allow quoted data
-      loop: true  # loop over in case of end-of-file reached
+      on-eof: loop  # what to do if end-of-file is reached.
```

Comment:
No, you can't eliminate the "loop" option, it is used by many existing users. I'd re-use the option name, having values
  - true = loop
  - false = not loop, stop the test

IMO the "continue" case has no practical sense